addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -81,78 +81,37 @@
 </template>
 
 <!-- Extend default mass_mailing snippets with website feature -->
-<template id="social_links">
-    <t t-if="not website">
-        <t t-set="website" t-value="request.env['website'].search([], limit=1)"/>
-    </t>
-    <t t-if="website.social_facebook">
-        <a t-att-href="website.social_facebook" aria-label="Facebook" title="Facebook">
-          <span class="fa fa-facebook"></span>
-        </a>
-    </t>
-    <t t-if="website.social_googleplus">
-        <a t-att-href="website.social_googleplus" style="margin-left:10px" aria-label="Google Plus" title="Google Plus">
-            <span class="fa fa-google-plus"></span>
-        </a>
-    </t>
-    <t t-if="website.social_linkedin">
-        <a t-att-href="website.social_linkedin" style="margin-left:10px" aria-label="LinkedIn" title="LinkedIn">
-            <span class="fa fa-linkedin"></span>
-        </a>
-    </t>
-    <t t-if="website.social_twitter">
-        <a t-att-href="website.social_twitter" style="margin-left:10px" aria-label="Twitter" title="Twitter">
-            <span class="fa fa-twitter"></span>
-        </a>
-    </t>
-    <t t-if="website.social_instagram">
-        <a t-att-href="website.social_instagram" style="margin-left:10px" aria-label="Instagram" title="Instagram">
-            <span class="fa fa-instagram"></span>
-        </a>
-    </t>
-</template>
-
-<template id="s_mail_block_header_social" inherit_id="mass_mailing.s_mail_block_header_social">
-    <xpath expr="//td[hasclass('o_mail_logo_container')]" position="after">
-        <td width="30%" class="text-right o_mail_no_resize">
-            <div class="o_mail_header_social">
-                <t t-call="website_mass_mailing.social_links"/>
-            </div>
-        </td>
-    </xpath>
-</template>
-
-<template id="s_mail_block_header_text_social" inherit_id="mass_mailing.s_mail_block_header_text_social">
-    <xpath expr="//table//td" position="after">
-        <td width="30%" class="text-right o_mail_no_resize">
-            <div class="o_mail_header_social">
-                <t t-call="website_mass_mailing.social_links"/>
-            </div>
-        </td>
-    </xpath>
-</template>
+<template id="website_social_links" name="Socila Link Template" inherit_id="mass_mailing.social_links">
+    <xpath expr="//div[@id='social_links']" position="replace">
```

Comment:
Noo :wink: xpath each individual link condition to add the new condition, the point of making this an inheriting view is to not duplicate the link definition in both apps.